VideoLocker/build.gradle

```diff
@@ -96,6 +96,7 @@ dependencies {
     compile 'com.android.support:cardview-v7:21.0.0'
     compile 'com.android.support:appcompat-v7:22.1.0'
     compile 'com.android.support:multidex:1.0.0'
+    compile 'com.github.bumptech.glide:glide:3.6.1'
```

Comment:
yeah, I plan too, but I'll do that in another commit, that wont be squashed with this one. So that in commit-history we know what, when and how it happened :wink: